libs.autobuild

```diff
@@ -20,6 +20,30 @@ cmake_package "drivers/glfw" do |pkg|
 end
 remove_from_default 'drivers/glfw'
 
+import_package 'drivers/imu_myahrs_plus' do |pkg|
+
+    def pkg.prepare
+        super
+        isolate_errors do
+            build
+            progress_done
+        end
+    end
+
+    def pkg.build
+        in_dir (File.join(srcdir, "common_cpp")) do
+            run("build", Autobuild.tool(:make))
+        end
+    end
+
+    def pkg.install
```

Comment:
Don't forget to call super in 'install', or you will end up re-building this package each time.